Dockerfile.amd64

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2015-2017 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM scratch
+LABEL maintainer "Casey Davenport <casey@tigera.io>"
+
+# Copy in the binary.
+ADD bin/confd-amd64 /bin/confd
```

Comment:
Does scratch have a PATH defined?  Not that we run this container at the mement but it seems desirable for `docker run calico/confd confd` to work.

Reply:
> Does scratch have a PATH defined?

Yes, quite. 

```
$ docker inspect calico/confd | jq
...
      "Env": [
        "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
      ],
...
```